release_notes/Service_Fabric_ReleaseNotes_90CU2.md

```diff
@@ -0,0 +1,96 @@
+|Service Fabric PowerShell and CLI | AzureRM PowerShell Module  <br> SFCTL |  0.3.15  <br> 11.0.1 |
+
+
+## Contents 
+
+Microsoft Azure Service Fabric 9.0 Cumulative Update 2.0 Release Notes
+
+* [Key Announcements](#key-announcements)
+* [Current Breaking Changes](#current-breaking-changes)
+* [Future Retirement and Deprecation Path Callouts](#future-retirement-and-deprecation-path-callouts)
+* [Service Fabric Common Bug Fixes](#service-fabric-common-bug-fixes)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+
+## Key Announcements
+* Windows Server 2022 is now supported as of the 9.0 CU2 release.
+* Mirantis Container runtime support on Windows for Service Fabric containers. To configure, see [Containers on Windows](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=dockerce#windows-server-1) documentation
+* The Microsoft Web Platform Installer (WebPI) used for installing Service Fabric SDK and Tools was retired on July 1, 2022. For latest guidance, see: [Set up Windows development environment](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-get-started#to-use-visual-studio-2017-or-2019)
+
+## Current Breaking Changes
+
+* **Guest Executable and Container Applications**: Guest executable and container applications created or upgraded in Service Fabric clusters with runtime versions 7.1+ are incompatible with prior Service Fabric runtime versions (e.g. Service Fabric 7.0).<br/>
+    Following scenarios are impacted:<br/>
+    * An application with guest executables or containers is created or upgraded in a Service Fabric 7.1+ cluster.<br/>
+    The cluster is then downgraded to a previous Service Fabric runtime version (e.g. Service Fabric 7.0).<br/>
+    The application fails to activate.<br/>
+    * A cluster upgrade from a version prior to Service Fabric v7.1 to Service Fabric 7.1+ version is in progress.<br/>
+    In parallel with the Service Fabric runtime upgrade, an application with guest executables or containers is created or upgraded.<br/>
+    The Service Fabric runtime upgrade starts rolling back (due to any reason) to a version prior to Service Fabric runtime v7.1.<br/>
+    The application fails to activate.<br/>
+    To avoid issues when upgrading from a version prior to Service Fabric runtime v7.1 to a Service Fabric 7.1+ runtime version, do not create or upgrade applications with guest executables or containers while the Service Fabric runtime upgrade is in progress.<br/>
+    * The simplest mitigation, when possible, is to delete and recreate the application in Service Fabric 7.0.<br/>
+    * The other option is to upgrade the application in Service Fabric 7.0 (for example, with a version only change).<br/>
+    If the application is stuck in rollback, the rollback has to be first completed before the application can be upgraded again.
+
+<br>
+
+## Future Retirement and Deprecation Path Callouts
```

Comment:
Please review this - we are copy pasting without consideration for what needs to be present vs not. For example, why do we have line items for entities expired in 2021 or SF 8.2? If the idea is to list timelines of all deprecations, we should remove "Future" from the heading and move the section to the end of the release notes. Otherwise, it breaks the flow for the content.

Reply:
The intention was to call out deprecations. I will move the section below "Service Fabric Common Bug Fixes" and change the header from "Future Retirement and Deprecation Path Callouts" to  "Retirement and Deprecation Path Callouts"
I will do the same for 8.2 CU4,9.0CU1,8.2CU3

Reply:
I wanted to remain consistent with the template that was followed in previous releases so Future Retirement and Deprecation Path Callouts remained before bug fixes.